ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.5-presubmits.yaml

```diff
@@ -14,6 +14,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-machine-config-operator-release-4.5-e2e-aws
     rerun_command: /test e2e-aws
+    run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
```

Comment:
not sure what this does (forgive my ignorance)

Reply:
I'm assuming you've taken this from the master branch so :+1:

Reply:
yeah, taken from master. I suspect this is perhaps to avoid running e2e-aws in scanrios where only platform specific changes has been made like. in openstack

Reply:
I think that's just marking doc changes as not necessary to run full CI